integration-tests/README.md

```diff
@@ -179,27 +193,43 @@ staticUnPrepare() - releases the cluster lease on wercker env.
 
 test methods -  testDomainOnPVUsingWLST, testDomainOnPVUsingWDT, testTwoDomainsManagedByTwoOperators, testCreateDomainWithStartPolicyAdminOnly, testCreateDomainPVReclaimPolicyRecycle, testCreateDomainWithDefaultValuesInSampleInputs, testAutoAndCustomSitConfigOverrides, testOperatorRESTIdentityBackwardCompatibility, testOperatorRESTUsingCertificateChain
 
-Utility classes:
+**Utility classes:**
 
-Operator - contains methods to create/destroy operator, verify operator created, scale using rest api, etc
-Domain - contains methods to create/destroy domain, verify domain created,deploy webapp, load balancing, etc
-PersistentVolume - to create PV
-LoadBalancer - to create load balancer
-Secret - to create secret
+Operator - constructor takes yaml file with operator properties and generates operator valus yaml with required properties and certs,  creates service account, namespace and calls helm install using the generated values yaml file. Also contains methods to delete operator release, verify operator created and ready, scale using rest api, verify a given domain via rest, verify external rest service, etc  <p>
+Domain - constructor takes Map with domain, LB, PV properties and creates domain crd, LB operator/ingress and PV artifacts using the sample scripts provided in the project. Also contains helper methods to destroy domain by deleting domain crd, verify domain created and servers are ready, deploy webapp, verify load balancing of http requests, etc <p>
+PersistentVolume - runs k8s job to create PV directory and creates PV and PVC using sample scripts  <p>
+LoadBalancer - creates load balancer, currently TREFIK and VOYAGER are supported <p>
+Secret - creates a k8s secret <p>
+TestUtils - mostly runs kubectl commands. Contains utility methods to check if a pod is created, ready, deleted, service created, get pod restart cnt, get cluster replica, delete PVC, check PV released, create rbac policy, create wldf module, etc. <p>
+ExecCommand - Class for executing shell commands from java <p>
+ExecResult - Class that holds the results of using java to exec a command (i.e. exit value, stdout and stderr) <p>
+K8sTestUtils - uses k8s java client api, this is used only for delete domain use cases for now. <p>
 
 # How to run the Java integration tests
 
 * Maven and latest Git should be in PATH
 * export JAVA_HOME
-
-Command to run the tests:
+* export WEBLOGIC_IMAGE_NAME and WEBLOGIC_IMAGE_TAG if different from store/oracle/weblogic and 12.2.1.3
```

Comment:
+ export IMAGE_PULL_SECRET_WEBLOGIC if a k8s secret is needed to access the image (plus deploy the k8s secret before running the test)

Reply:
@tbarnes-us secret IMAGE_PULL_SECRET_WEBLOGIC is created using the docker credentials provided by DOCKER_USERNAME, DOCKER_PASSWORD, DOCKER_EMAIL. Its not required to export IMAGE_PULL_SECRET_WEBLOGIC as the test takes care of that.

Reply:
@tbarnes-us secret IMAGE_PULL_SECRET_WEBLOGIC is created using the docker credentials provided by DOCKER_USERNAME, DOCKER_PASSWORD, DOCKER_EMAIL. Its not required to export IMAGE_PULL_SECRET_WEBLOGIC as the test takes care of that.

Reply:
Oh - I see.  Isn't it possible that it is required when USER/PASS/EMAIL are not specified?   I should have made that clear.

Reply:
Currently tests expect USER/PASS/EMAIL if the image doesn't exist locally, otherwise it fails with the error message asking to provide the docker login details.